src/test/java/com/hotels/mutantswarm/exec/OutcomeTest.java

```diff
@@ -39,11 +40,19 @@
   @Mock
   private Mutation mutation;
   @Mock
+  private Mutation mutation2;
+  @Mock
   private TestOutcome testOutcome1, testOutcome2, testOutcome3; 
   
+  private List<TestOutcome> testOutcomes;
+  
+  @Before
+  public void setUpMocks() {
```

Comment:
```suggestion
  public void setUp() {
```
Might as well make the method name generic as it's not really doing anything mock-specific.